### PR TITLE
[FSDE-1206] Add support for portal theme

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSDE-1206

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-114/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-114/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-114/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1e7ff8ccac19ab89e2e8be52271ad4920dee6110-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-114/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d6aa7d80d7429ea21536103898387d083493c062-chi-pr.enterprise.ctl.io

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: src/website/views/components/sidenav/_examples.pug [hunk 1]
```diff
@@ -10,242 +10,243 @@
...
+        </aside>
 
-      <script>
-        chi.sidenav(document.getElementById('sidenav-0'));
-      </script>
+        <script>
+          chi.sidenav(document.getElementById('sidenav-0'));
+        </script>
 
 h3 Open on hover
```
[the remark this answers is not in the document]
Comment (mattnickles) [reply src/website/views/components/sidenav/_examples.pug:251]: Great catch! Thanks

File: src/website/views/components/tooltip/_examples.pug [hunk 2]
```diff
@@ -31,36 +31,37 @@ p.-text
...
-      span.-text--grey
-        | This HTML Blueprint requires JavaScript.
-        | You may use your own solution, or use Chi's vanilla JavaScript solution,
-        | <a href="../../getting-started/installation">Chi.js</a>.
-    :code(lang='html')
-      <button id="tooltip-2" class="chi-button -outline -light" data-tooltip="Your tooltip text on a button" data-tooltip-color="light">Hover me to see tooltip</button>
+.-lumen-only.-centurylink-only
+  h3 Light
+  p.-text
+    | Use light tooltips for rendering tooltips on dark backgrounds. To use, apply the <code>data-tooltip-color="light"</code> attribute to the Tooltip trigger element.
```
[the remark this answers is not in the document]
Comment (mattnickles) [reply src/website/views/components/tooltip/_examples.pug:37]: I will move them. Thanks!

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-114/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f2fffb3b3655c9aa0c9451d4356d928f819842e2-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by jllr** [comment]
@mattnickles great job!!

**Pull request merged**